### PR TITLE
Docs: HL Burrow Documentation and RTD Fixes

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

Added documentation for HL Burrow based on the documentation by @ekovalev : general info and API
Fixed table wrapping
Added lexer for Solidity code

### Benefits

Will be ready for HL Burrow code
Better looking docs

### Possible Drawbacks 

Table wrap might might not look right sometimes
New requirement for the docs (lexer)
There are still questions in possible API errors

### CODE REVIEW

**Comment by LiraLemur** [comment]
@iceseer @MBoldyrev I believe there were some changes in how this works. Can you please explain those to me so I could fix it?

File: docs/source/develop/api/queries.rst [hunk 1]
```diff
@@ -15,6 +15,89 @@ The validation for all queries includes:
...
+
+Response Schema
+---------------
+
+.. code-block:: proto
+
+    message EngineReceiptsResponse {
+        repeated EngineReceipt engine_receipt = 1;
+    }
+    message EngineReceipt {
```
Comment (ekovalev) [inline docs/source/develop/api/queries.rst:54]: This interface doesn't match the proto anymore after changes in the latter. Needs to be aligned with the source.

Comment (LiraLemur) [reply docs/source/develop/api/queries.rst:54]: @ekovalev  can you please check now?

File: docs/source/integrations/burrow.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+From the Burrow EVM perspective such Service Contract is hosted in a Native external VM and is callable via the same interfaces as if it was deployed at some special address in the EVM itself.
+
+.. note:: You can check out `Burrow documentation <https://wiki.hyperledger.org/display/burrow/Burrow+-+The+Boring+Blockchain>`_ for more information on Natives and external dispatchers.
+
+Schematically the interaction between different parts of the system looks as follows:
+
+.. image:: ../../image_assets/burrow/natives.png
+
+Current release of the Iroha EVM wrapper contains a single service contract deployed at the address `A6ABC17819738299B3B2C1CE46D55C74F04E290C` (the last 20 bytes of the *keccak256* hash of the string *ServiceContract*) which exposes 2 methods to query Iroha assets balances and transfer assets between accounts.
+The signature of these four method looks like this:
```
Comment (lebdron) [inline docs/source/integrations/burrow.rst:65]:
```suggestion
The signatures of these two methods look like this:
```

**Pull request merged**